src/rules/padding-before-describe-blocks.ts

```diff
@@ -1,48 +1,14 @@
-/**
- * @fileoverview Enforces single line padding around describe blocks
- * @author Dan Green-Leipciger
- */
-'use strict';
-
-import { padBefore } from '../utils';
-//------------------------------------------------------------------------------
-// Rule Definition
-//------------------------------------------------------------------------------
-
-const beforeMessage =
-  'You need a newline or comment before a describe block when it comes after another expression';
-
-const isDescribe = node =>
-  node.expression &&
-  node.expression.callee &&
-  node.expression.callee.name === 'describe';
+import padding from './padding';
 
 export default {
-  meta: {
-    docs: {
-      description: 'Enforces at least a line of padding before describe blocks',
-      category: 'Fill me in',
-      recommended: false,
-    },
-    fixable: 'whitespace', // or "code" or "whitespace"
-    schema: [
-      // fill in your schema
-    ],
-  },
-  beforeMessage,
+  ...padding,
   create(context) {
-    const filePath = context && context.getFilename();
-    const isTest =
-      filePath.includes('test') ||
-      filePath.includes('spec') ||
-      filePath.includes('<input>');
-    return {
-      ExpressionStatement(node) {
-        if (!isTest || !isDescribe(node)) {
-          return;
-        }
-        padBefore({ context, node, beforeMessage });
-      },
-    };
+    const ctx = Object.create(context, {
+      options: {
+        value: [{blankLine: 'always', prev: '*', next: 'describe'}]
+      }
+    });
+
+    return padding.create(ctx);
```

Comment:
After altering the config we call the original padding.create with the new context.